plugins/parsers/logfmt/parser_test.go

```diff
@@ -21,7 +21,7 @@ func TestParse(t *testing.T) {
 	}{
 		{
 			name: "no bytes returns no metrics",
-			want: []telegraf.Metric{},
+			want: make([]telegraf.Metric, 0),
```

Comment:
Remove init? Ideally we should split the test into cases producing metrics and tests not producing metrics but that's probably too much to ask for this PR... ;-)

Reply:
Removed